### ISSUE TITLE
merge LR bam files by TieBrush

### ISSUE THREAD

**Issue by YIGUIz** [gpertea/tiebrush#6]
Hi, 

I recently used TieBrush to merge BAM files and noticed that it worked effectively on short-read BAM files. However, it didn't seem to reduce the read count for long-read BAM files.

LR merge:
![image](https://github.com/user-attachments/assets/d5d54a42-14cb-41c4-86eb-16e4e12bcfb5)

SR merge:
![image](https://github.com/user-attachments/assets/e4786c62-89e2-4fd6-ad3c-ad63716a2872)

Could you kindly advise if the software is primarily intended for short-read data, or if there are specific parameters I can adjust to merge long-read BAM files?

Thank you for your assistance. I look forward to your guidance.

Best regards,
Qi

**Comment by gpertea** [comment]
Indeed, Tiebrush was devised for and tested on short-read data only. What you observed is expected, due to the nature of long read alignments, with large and complex CIGAR strings due to higher error rate and much higher chance of having some alignment variations/errors between reads aligned to the same isoform. 

One thing you could try is using the `-E` option with tiebrush with long reads, which ignores some of these small alignment variations/errors and only consider the rough exon/intron coordinates, though we noticed that intron coordinates (splice junction locations) can also be a bit "fuzzy" in the case of long read alignments.

**Comment by YIGUIz** [comment]
Thanks for your suggestions!
 
I’m planning to assemble a tissue transcriptome using StringTie2 (-mix). I have about 400 long-read RNAseq datasets and paired short-read RNAseq data. Do you have any ideas on how to merge all the BAM files to ensure the program runs smoothly?

**Comment by YIGUIz** [comment]
Thanks for your suggestions!

I’m planning to assemble a tissue transcriptome using StringTie2 (-mix). I have around 400 long-read RNAseq samples, along with paired short-read RNAseq data. Do you have any recommendations for merging all the BAM files to ensure smooth execution?

Also, I have another question: what’s the difference between assembling by sample (assembling each sample individually and then merging the GTF files) and assembling by tissue (merging all BAM files into one before assembly)? Why does CHESS3 choose overlapping transcripts?
